server.js

```diff
@@ -36,8 +37,7 @@ app.get("/", function(req, res) {
     return;
   }
   res.render("index", {
-    // only latest 10 mail
-    data: store.slice(0, 10)
+    data: store.slice(0, fetchLimit)
```

Comment:
store data length is at most `fetchLimit `.
Because you may not use slice.

Reply:
Yes, `slice` is unnecessary. I'll delete it.
Thank you.